### PR TITLE
Fix bst auditory

### LINKED ISSUE #3869: BUG: plot_brainstorm_auditory.py example broken
For some reason the bst_auditory `stc_difference` plot looks bad on [dev builds](https://3360-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_tutorials/plot_brainstorm_auditory.html) compared to [stable](https://mne-tools.github.io/stable/auto_tutorials/plot_brainstorm_auditory.html).

### PR DESCRIPTION
**Pull request by agramfort**
fixes #3869

### CODE REVIEW

**Comment by Eric89GXL** [comment]
You have merge conflicts already :)

File: tutorials/plot_brainstorm_auditory.py [hunk 1]
```diff
@@ -258,7 +258,7 @@
     notches = [60, 120, 180]
     for evoked in (evoked_std, evoked_dev):
         evoked.data[:] = notch_filter(evoked.data, sfreq, notches)
-        evoked.data[:] = filter_data(evoked_std.data, sfreq, None, 100)
```
Comment (agramfort) [inline tutorials/plot_brainstorm_auditory.py]: this was the bug...

Comment (Eric89GXL) [inline tutorials/plot_brainstorm_auditory.py]: Looks like I broke it back in October :(

File: mne/viz/misc.py [hunk 2]
```diff
@@ -111,8 +111,7 @@ def plot_cov(cov, info, exclude=[], colorbar=True, proj=False, show_svd=True,
     fig_cov, axes = plt.subplots(1, len(idx_names), squeeze=False,
                                  figsize=(2.5 * len(idx_names), 2.7))
     for k, (idx, name, _, _) in enumerate(idx_names):
-        axes[0, k].imshow(C[idx][:, idx], interpolation="nearest",
-                          cmap='RdBu_r')
+        axes[0, k].imshow(C[idx][:, idx], interpolation="nearest", cmap='RdBu_r')
```
Comment (Eric89GXL) [inline mne/viz/misc.py]: this is a PEP8 violation

**Comment by agramfort** [comment]
good to go

**Comment by Eric89GXL** [comment]
Thanks @agramfort

**Pull request merged**